util/mkinstallvars.pl

```diff
@@ -10,18 +10,24 @@
 # form, or passed as variable assignments on the command line.
 # The result is a Perl module creating the package OpenSSL::safe::installdata.
 
+use 5.10.0;
+use strict;
+use warnings;
+use Carp;
+
 use File::Spec;
-use List::Util qw(pairs);
+#use List::Util qw(pairs);
+sub _pairs (@);
 
 # These are expected to be set up as absolute directories
 my @absolutes = qw(PREFIX libdir);
 # These may be absolute directories, and if not, they are expected to be set up
 # as subdirectories to PREFIX or LIBDIR.  The order of the pairs is important,
 # since the LIBDIR subdirectories depend on the calculation of LIBDIR from
 # PREFIX.
-my @subdirs = pairs (PREFIX => [ qw(BINDIR LIBDIR INCLUDEDIR APPLINKDIR) ],
-                     LIBDIR => [ qw(ENGINESDIR MODULESDIR PKGCONFIGDIR
-                                    CMAKECONFIGDIR) ]);
+my @subdirs = _pairs (PREFIX => [ qw(BINDIR LIBDIR INCLUDEDIR APPLINKDIR) ],
```

Comment:
Why do it with `_pairs` when you can just type 2 pairs of `[` / `]`?

array-refs have been core in perl for some time

Reply:
Exactly, this is just a very round-about way of writing them as literal array refs, since you're only supplying 4 items to `pairs` or `_pairs`.

Reply:
Good point.  I had done it like this initially 'cause I found it more elegant, simple as that.

But, now it's in with `_pairs`.  If someone is really hell bent on changing this bit *again*, feel free to raise a PR.